package.json

```diff
@@ -89,6 +96,13 @@
       }
     ],
     "menus": {
+      "view/item/context": [
+        {
+          "_comment": "When model file is right-cliched, [Compile] menu will be shown.",
+          "command": "onevscode.show-compile-webview",
+          "when": "view == FileTreeView && viewItem == compilableFile"
```

Comment:
Refer to the following for `compilableFile`:

https://github.com/hyunsik-yoon/ONE-vscode/blob/2b8b2af0c37f469ca8c688da030cbb29027f9e43/src/Views/FileTreeView.ts#L373-L389